packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1370,7 +1368,9 @@ export class AccountHandler {
           uid: account.uid,
           generation: account.verifierSetAt,
         }),
-        this.oauth.removePublicAndCanGrantTokens(account.uid),
+        // `removeUser` removes the account's oauth tokens and authorization
+        // codes.  it does not remove the user.
+        this.oauth.removeUser(account.uid),
```

Comment:
nit: Thanks for the comment, maybe we could refactor the function name to reflect what it does.